distribution/bwc/build.gradle

```diff
@@ -322,6 +322,22 @@ BuildParams.bwcVersions.forPreviousUnreleased { BwcVersions.UnreleasedVersionInf
     }
     // make sure no dependencies were added to assemble; we want it to be a no-op
     assemble.dependsOn = []
+
+    // expose this method to allow building a test plugin for an old cluster (see qa/task-cancellation)
+    project.ext.createBwcGradleTask = { taskName, gradleTask ->
```

Comment:
We don't want other projects creating tasks under the `:distribution:bwc` project. If you want an example of how this might be done take a look at [this PR](https://github.com/elastic/elasticsearch/pull/59322/files) which is quite similar in that it is building the JDBC jar for BWC branches.

Reply:
++. I've adopted in https://github.com/elastic/elasticsearch/pull/59828/commits/343213861d4af740d3fd4c0ee076170fe866f943.